src/uiTest/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -18,7 +18,6 @@ trait RunningIntelliJPerSuite extends RunningIntelliJPerSuiteBase {
   private val ignoredErrorMessages = Seq(
     // Spurious errors in the IDEA itself (probably some race conditions)
     "com.intellij.diagnostic.PluginException: Cannot create class com.intellij.uast.UastMetaLanguage",
-    "com.intellij.serviceContainer.AlreadyDisposedException: Already disposed: Project",
     "at com.intellij.util.messages.impl.MessageBusConnectionImpl.deliverImmediately(MessageBusConnectionImpl.java:61)"
```

Comment:
Actually... maybe this got sorted out as well with #811, I'll remove that on a separate PR for experiment